DevoxxClientMobile/src/main/java/com/devoxx/views/layer/ConferenceLoadingLayer.java

```diff
@@ -0,0 +1,90 @@
+package com.devoxx.views.layer;
+
+import com.devoxx.service.Service;
+import com.gluonhq.charm.down.Services;
+import com.gluonhq.charm.down.plugins.DisplayService;
+import com.gluonhq.charm.glisten.application.GlassPane;
+import com.gluonhq.charm.glisten.application.MobileApplication;
+import com.gluonhq.charm.glisten.control.ProgressIndicator;
+import com.gluonhq.charm.glisten.layout.Layer;
+import javafx.animation.PauseTransition;
+import javafx.beans.InvalidationListener;
+import javafx.beans.Observable;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.TextAlignment;
+import javafx.util.Duration;
+
+public class ConferenceLoadingLayer extends Layer {
+    
+    private static Duration TIMEOUT = Duration.seconds(20);
+
+    private final GlassPane glassPane;
+    private final ProgressIndicator progressIndicator;
+    private final Label conferenceLabel;
+    private Region background;
+    private final DisplayService displayService;
+
+    private ConferenceLoadingLayer(Service service, String conferenceName) {
+
+        this.glassPane = MobileApplication.getInstance().getGlassPane();
+        progressIndicator = new ProgressIndicator();
+
+        conferenceLabel = new Label(conferenceName);
+        conferenceLabel.setWrapText(true);
+        conferenceLabel.setTextAlignment(TextAlignment.CENTER);
+        conferenceLabel.setFont(Font.font(Font.getDefault().getFamily(), FontWeight.BOLD, 15));
```

Comment:
Font is too small on tablet.
Maybe this should be moved (later on?) to CSS.

Reply:
I have set it to 1.3em on `common_tablet.css`